breathe/renderer/sphinxrenderer.py

```diff
@@ -389,7 +426,13 @@ def get_node_info(file_data):
         def render_signature(file_data, doxygen_target, name, kind):
             # Defer to domains specific directive.
             self.context.directive_args[1] = [self.get_fully_qualified_name()]
-            nodes = self.run_domain_directive(kind, self.context.directive_args[1])
+
+            if kind in ('class', 'struct'):
+                augment = dict(bases=file_data.compounddef.basecompoundref)
```

Comment:
Do `interface`s have base classes too?

Reply:
Don't believe so, this is probably fine.